content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/index.md

```diff
@@ -0,0 +1,178 @@
+
+Like any other event, you need to register at their official site. You can learn about upcoming events on their [official page](https://www.cncf.io/events/). Sometimes it might be not easy to afford the cost of attending these events, but CNCF also offers scholarships to individuals. To provide an overview, CNCF mainly offers scholarships based on 3 categories namely - Diversity, Need-Based, and Maintainer.
+
+<img src="pic2.png" alt="Scholarship Details">
+
+KubeCon serves as a dynamic platform for attendees to actively engage with the expansive Kubernetes community. Through a diverse array of networking avenues, including Birds of a Feather sessions, social events, and dedicated breaks, participants have the opportunity to connect with fellow developers, contributors, and users of Kubernetes.
+
+These interactions foster collaboration and facilitate the exchange of knowledge and experiences. The conference's emphasis on networking creates an environment where like-minded professionals and industry experts can forge connections, share insights, and collectively contribute to the ever-evolving landscape of container orchestration and cloud-native technologies.
+
+<h2>General Information about KubeCon</h2>
+
+KubeCon North America 2023 was held from the 6th to the 9th of November 2023, in Chicago, Illinois.
+
+<img src="pic3.jpg" alt="Cloud Gate, Chicago">
+
+Education and collaboration are vital to the future of the CNCF ecosystem. It's imperative to CNCF that everyone in the community who wants to participate feels welcome, regardless of gender, gender identity, sexual orientation, dis/ability, race, ethnicity, age, religion, or economic status.
+
+<p align="center">
+  <img src="pic4.jpg" alt="Priyanka Sharma KubeCon">
+</p>
+</br>
+
+Every KubeCon has more or less the same schedule except for the talks. The talks vary depending on the [CFPs](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/program/cfp/) submitted. To give a very basic plot of the event, This was the schedule of KubeCon 2023:
+
+<img src="pic5.png" alt="KubeCon Schedule">
+</br>
+
+In case you want to check the full schedule, click [here](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/schedule/). You can change the timezone of the schedule according to your local time.
+
+For a beginner, attending KubeCon can be overwhelming. Some might consider leaving tech and open-source. But hold your nerve. There's a place for everyone in the community. As a beginner, I'd suggest carefully examining the schedule for talks aligned with your interests. Keynotes, mainly by CNCF Ambassadors, are crucial for beginners.
+
+<h2>My experiences</h2>
+
+I want to showcase some of the talks I attended at KubeCon North America virtually.
+</br>
+<img src="ss1.png">
+</br>
+**Keynote: Welcome + Opening Remarks - Priyanka Sharma, Executive Director, Cloud Native Computing Foundation**
```

Comment:
Please use Markdown - https://www.digitalocean.com/community/tutorials/markdown-markdown-images has an example. Or a `figure` shortcode if you want to caption it.